### DISCUSSION TITLE
Couple to feature requests/questions about Correspondent, Tags, Recipient

### DISCUSSION THREAD

**Discussion by Sky-Dragon** [jonaswinkler/paperless-ng#848] [General]
Hello all,
I have been looking for a DMS for a long time and finally found the right one for me, thanks for that.

I hope it's not too many questions/suggestions etc in one disskusion, otherwise please split it or I'll split it.
While migrating I found that a few features could be helpful.
1) Correspondent names in short and original name.
2) Parent-child relationship in TAGS
3) Recipient to if the document is me or my wife (not client system)

to 1)
May also overlap with #274 ?
In Paperless I would like to maintain both names, with the background, if the documents are exported with [PAPERLESS_FILENAME_FORMAT](https://paperless-ng.readthedocs.io/en/latest/advanced_usage.html#advanced-file-name-handling), that there is only the short name.
In Paperless itself you see both names, so that it is also complete.
e.g.
Short name: DRV
Original: Deutsche Rentenversicherung Westfalen
Short name: XYZ-NAME
Original: Hausverwaltung & Immobilien XYZ-NAME GmbH

To 2)
Since I made my filing up to now on folder basis, I put on a structure that also others understand/understood fast. Therefore I have from it the appropriate thought.
The TAGS could be represented then as "parent-child" and be extended accordingly also that the child becomes a parent relationship.
e.g.
Parents: Insurance
Child: Life
Child: Car
-->     Insurance-Life                  Insurance-Car
or --> Insurance/Life                  Insurance/Car

Parents: Tax
Child: 2020
Child: 2021

To 3)
Since my wife and I each have auto insurance with the same insurer (or bank), this makes it quicker to identify when the document is addressed to. i don't see it in the metadata so it works with PAPERLESS_FILENAME_FORMAT.

I apologize for my bad english already, unfortunately this is not my native language :-)

Greetings 
Alex

**Comment by shamoon** [comment]
Hi @Sky-Dragon ,

I think 2 is covered by #56 which is on the roadmap but not a small change. For your third item, I have a similar situation and created tags for myself and others which I think is a pretty good solution for now but yes I also think custom metadata will be useful for that.

**Reply by Sky-Dragon** [reply]
Hi @shamoon,
to 2)
thanks for #56 that sounds good, that's exactly what I was thinking, if one is selected, all others should be selected as well. In addition to a your extension (can't find it currently) where to exclude certain tags.
So you could show all e.g. accounts of the bank, but not a specific one.
I hope that this extension comes soon, until then I go the intermediate way, that I create each child tag individually.
The #88 linked in #56 is even a step further than I had previously thought, I like that in a community - one head is good many are better!

to 3)
Current I do that also, that I take up the person additionally as a tag. But I have to admit that I try to use as few tags as possible, so that when exporting to a file structure it still remains human readable and structured (PAPERLESS_FILENAME_FORMAT).
With PAPERLESS_FILENAME_FORMAT I haven't experimented much yet, because there was/is a bug in the old version that I could only do this once and changes to PAPERLESS_FILENAME_FORMAT then had no effect and I didn't want to break my current work. Should be #750. 
I hope that the metadata is also exportable :-) .
